rotors_gazebo_plugins/src/gazebo_wind_plugin.cpp

```diff
@@ -24,6 +24,9 @@
 #include <geometry_msgs/Vector3.h>
 #include <geometry_msgs/WrenchStamped.h>
 
+#include <math.h>
```

Comment:
I understand. I changed the order, and will modify the if/for statements according to the style guidelines.